python/AccessPROBA-V/PROBA_V.ipynb

```diff
@@ -0,0 +1,1293 @@
+    "for collection in collections:\n",
+    "    if \"title\" in collection and \"PROBA-V\" in collection[\"title\"]:\n",
+    "        print(f\"ID: {collection.get('id')}, Title: {collection.get('title')}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "58f74bfd",
+   "metadata": {},
+   "source": [
+    "Users can obtain detailed descriptions of each collection by executing the command `connection.describe_collection(\"PROBAV_L3_S10_TOC_333M\")`. \n",
+    "\n",
+    "For this example, we will use the `PROBAV_L3_S5_TOC_100M` collection.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "id": "bfccd1e4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load data cube from PROBAV_L3_S5_TOC_100M collection.\n",
+    "cube = connection.load_collection(\n",
+    "    \"PROBAV_L3_S5_TOC_100M\",\n",
+    "    temporal_extent=[\"2021-03-01\", \"2021-06-01\"],\n",
+    "    bands=[\"NDVI\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "cd61f53e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# apply offset and rescale\n",
+    "cube = cube.apply(lambda x: 0.004 * x - 0.08)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e1f3c1f1",
+   "metadata": {},
+   "source": [
+    "Users can either download the NDVI data retrieved in the previous cell or proceed with further analysis. \n",
+    "\n",
+    "Here, we will create an NDVI time series. In the following cell, we will use the `DataCube.aggregate_spatial()` method to compute the mean NDVI for each specified field."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "38e1fcfe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def read_json(filename: str) -> dict:\n",
+    "    with open(filename) as input:\n",
+    "        field = json.load(input)\n",
+    "    return field\n",
+    "\n",
+    "\n",
+    "fields = read_json(\"polygons.geojson\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "a9fb4cd7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ndvi_timeseries = cube.aggregate_spatial(geometries=fields, reducer=\"mean\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aa8a8da4",
+   "metadata": {},
+   "source": [
+    "Finally, let us perform the computation synchronously and retrieve the results as a CSV file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "211da31a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ndvi_timeseries.download(\"ProbaV10days.csv\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "47bb2a97",
+   "metadata": {},
+   "source": [
+    "With the above result, let us visualizes the distribution of NDVI (Normalized Difference Vegetation Index) values over time using a box plot. This plot helps in understanding how vegetation health evolves over time "
```

Comment:
this is an odd sentence

In the following cell we will visualize...

Apart from that the visualization is not rendered

Reply:
I still do not see the visualisation in the notebook?

Reply:
I have included the output as a png. However, the script was also included to generate it dynamically. It could be possible that it wasn't rendered earlier due to the absence of some issues. But it should be fine in the Terrascope Jupyter environment.